### ISSUE TITLE
Fix: `Error: -exec [args...] + isn't supported yet. Only -exec [args...] ;`

### ISSUE THREAD

**Issue by sylvestre** [uutils/findutils#366]
`Error: -exec [args...] + isn't supported yet. Only -exec [args...] ;`
example:
` find . ! -newermt 'jan 01, 2000' -exec touch -d@1714807765 {} +`

**Comment by hanbings** [comment]
Looks like what is mentioned at https://github.com/uutils/findutils/issues/6. 
In addition to `exec`, it is also necessary to support the `+` syntax of `execdir`.

**Comment by winstonma** [comment]
Yep I faced this issue after replacing findutils through [oxidizr](https://github.com/jnsgruk/oxidizr) in Ubuntu 24.10:

This is a snippet of the script that uses `find` in `/usr/share/initramfs-tools/hooks/udev`
```
...
find -L /usr/lib/systemd/network -name '*.link*' -execdir cp -r -pt "$DESTDIR/usr/lib/systemd/network/" '{}' +
...
```

**Comment by hanbings** [comment]
> Yep I faced this issue after replacing findutils through [oxidizr](https://github.com/jnsgruk/oxidizr) in Ubuntu 24.10:
> 
> This is a snippet of the script that uses `find` in `/usr/share/initramfs-tools/hooks/udev`
> 
> ```
> ...
> find -L /usr/lib/systemd/network -name '*.link*' -execdir cp -r -pt "$DESTDIR/usr/lib/systemd/network/" '{}' +
> ...
> ```

I'm very sorry for spending so much time on this issue. I'll do my best to complete it within two weeks. If you have any ideas for solving it, feel free to discuss them with me — PRs are also welcome. :)

**Comment by winstonma** [comment]
I'm posting this because whenever an error occurs on their system, they seem to be aware of the root cause, at least in my experience. The error typically happens when the udev script is executed after a new kernel is installed or when [Plymouth](https://wiki.ubuntu.com/Plymouth) is being upgraded. 

When the issue arises, they appear to know that a short-term solution is to reinstall the original findutils package.
```
#METHOD #1
sudo oxidizr disable --experiments findutils

# METHOD #2
sudo apt install --reinstall findutils
```
